src/update-all/update-all-dialog.ts

```diff
@@ -1,24 +1,42 @@
 import { LitElement, html } from "lit";
-import { customElement } from "lit/decorators.js";
+import { customElement, property } from "lit/decorators.js";
 import "@material/mwc-button";
-import "../components/remote-process";
-import "../components/process-dialog";
+import "@material/mwc-dialog";
+import { openUpdateAllProcessDialog } from "./update-all-process";
 
 @customElement("esphome-update-all-dialog")
 class ESPHomeUpdateAllDialog extends LitElement {
+  @property() public name!: string;
+  @property() public configuration!: string;
+
   protected render() {
     return html`
-      <esphome-process-dialog
-        heading="Update All"
-        .type=${"update-all"}
-        @closed=${this._handleClose}
-      ></esphome-process-dialog>
+      <mwc-dialog heading="Update All" @closed=${this._handleClose} open>
```

Comment:
Please create a generic confirm dialog that works with promises.

```ts
if (await confirm("Update All", "Do you want to update all devices", "Update All")) {
    ...
}
```

And then call that from the openUpdateAllDialog function

Reply:
I think that this is a bit beyond my knowledge, but I'll search in HA Frontend repo and maybe will be able to propose something.

Reply:
done